tools/jaeger/jaeger-config.yaml

```diff
@@ -0,0 +1,82 @@
+# Enable the spanmetrics connector to bridge
+# the traces pipeline into the metrics pipeline.
+service:
+  extensions: [jaeger_storage, jaeger_query, healthcheckv2]
+  pipelines:
+    traces:
+      receivers: [otlp]
+      processors: [batch]
+      exporters: [jaeger_storage_exporter, spanmetrics]
+    metrics/spanmetrics:
+      receivers: [spanmetrics]
+      exporters: [prometheus]
```

Comment:
YAML config file for Jaeger local container. Jaeger receives the **traces** (forwarded by otel-collector) and also reads metrics generated by Prometheus to show a UI where you can search/list the traces and also show metrics like the ones below:
![pims-RED-metrics](https://github.com/user-attachments/assets/03570db4-677a-4fc3-9fb5-1d66bb9f70dc)